### PR TITLE
1.0: update docs about reset task and relay log purge

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

update the FAQ about reset task
clarify the purge-expires config item

### Which DM version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v2.0 changes for now)(master is updated in https://github.com/pingcap/docs-dm/pull/180)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/104
- Other reference link(s): the same purpose as #103 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/faq.md [hunk 1]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+3. Copy the normal binlog file from the upstream MySQL to replace the corresponding file in the [relay log directory](relay-log.md#directory-structure) of DM-worker.
+
+    - If the cluster is deployed using DM-Ansible, the relay log is in the `<deploy_dir>/relay_log` directory.
+    - If the cluster is manually deployed using the binary, the relay log is in the directory set of the `relay-dir` parameter.
+
+4. Modify the information of `relay.meta` in the relay log directory of DM-worker to the information corresponding to the next binlog file.
+
+    - If `enable-gtid` is not enabled, set `binlog-name` to the file name of the next binlog file, and set `binlog-pos` to `4`. If you copy `mysq-bin.000100` from the upstream MySQL to the relay directory, and want to continue to pull binlog from `mysql-bin.000101` later, set `binlog-name` to `mysql-bin.000101` `. 
+
+    - If `enable-gtid` is enabled, set `binlog-gtid` to the value corresponding to `Previous_gtids` at the beginning of the next binlog file. You can obtain the value by executing [SHOW BINLOG EVENTS](https:// dev.mysql.com/doc/refman/5.7/en/show-binlog-events.html).
```
Comment (csuzhangxc) [inline en/faq.md:65]:
```suggestion
    - If `enable-gtid` is enabled, set `binlog-gtid` to the value corresponding to `Previous_gtids` at the beginning of the next binlog file. You can obtain the value by executing [SHOW BINLOG EVENTS](https://dev.mysql.com/doc/refman/5.7/en/show-binlog-events.html).
```

File: en/faq.md [hunk 2]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+
+5. Refer to [restart DM-worker](cluster-operations.md#restart-dm-worker) to **start** the abnormal DM-worker node.
+
+6. Use `start-task` to resume a stopped migration task.
```
Comment (csuzhangxc) [inline en/faq.md:69]:
```suggestion
6. Use `start-task` to resume all stopped migration tasks.
```

File: en/faq.md [hunk 3]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+
+#### The required relay log has been cleaned up in upstream MySQL
```
Comment (csuzhangxc) [inline en/faq.md:71]:
```suggestion
#### The required relay log has been purged in upstream MySQL
```
? `purge` is used in MySQL docs.

File: en/faq.md [hunk 4]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+
+If the relay log required by the migration task is abnormal in the DM-worker, and has been cleaned up in the upstream MySQL, you can use the following steps to restore the data migration task:
```
Comment (csuzhangxc) [inline en/faq.md:73]:
```suggestion
If the relay log required by the migration task is abnormal in the DM-worker, and has been purged in the upstream MySQL, you can use the following steps to restore the data migration task:
```

File: en/faq.md [hunk 5]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
 
-## How to reset the data replication task?
+## How to reset the data migration task?
 
-You need to reset the entire data replication task in the following cases:
+### Reset the data migration task when the relay log is in the normal state
 
-- `RESET MASTER` is accidentally executed in the upstream database.
-- The relay log or the upstream binlog is corrupted or lost.
+If the relay log required by the data migration task is normal, you can use the following steps to restore the data migration task:
```
Comment (ran-huang) [inline en/faq.md:35]:
```suggestion
If the relay log required by the data migration task is normal, you can use the following steps to reset the data migration task:
```

File: en/faq.md [hunk 6]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+If the relay log required by the migration task is abnormal in the DM-worker, but is normal in the upstream MySQL, you can use the following steps to restore the data migration task:
+
+1. Use the `stop-task` command to stop all the migration tasks that are currently running. 
+
+2. Refer to [restart DM-worker](cluster-operations.md#restart-dm-worker) to **stop** the abnormal DM-worker node.
+
+3. Copy the normal binlog file from the upstream MySQL to replace the corresponding file in the [relay log directory](relay-log.md#directory-structure) of DM-worker.
+
+    - If the cluster is deployed using DM-Ansible, the relay log is in the `<deploy_dir>/relay_log` directory.
+    - If the cluster is manually deployed using the binary, the relay log is in the directory set of the `relay-dir` parameter.
```
Comment (ran-huang) [inline en/faq.md:59]:
```suggestion
    - If the cluster is manually deployed using the binary, the relay log is in the directory set by the `relay-dir` parameter.
```

File: en/faq.md [hunk 7]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+
+    - If `enable-gtid` is enabled, set `binlog-gtid` to the value corresponding to `Previous_gtids` at the beginning of the next binlog file. You can obtain the value by executing [SHOW BINLOG EVENTS](https://dev.mysql.com/doc/refman/5.7/en/show-binlog-events.html).
+
+5. Refer to [restart DM-worker](cluster-operations.md#restart-dm-worker) to **start** the abnormal DM-worker node.
+
+6. Use `start-task` to resume all stopped migration tasks.
+
+#### The required relay log has been purged in upstream MySQL
+
+If the relay log required by the migration task is abnormal in the DM-worker, and has been purged in the upstream MySQL, you can use the following steps to restore the data migration task:
```
Comment (ran-huang) [inline en/faq.md:73]:
```suggestion
If the relay log required by the migration task is abnormal in the DM-worker, and has been purged in the upstream MySQL, you can use the following steps to reset the data migration task:
```

File: en/faq.md [hunk 8]
```diff
@@ -28,25 +28,67 @@ When you encounter a DDL statement unsupported by TiDB, you need to manually han
...
+
+1. Use the `stop-task` command to stop all the migration tasks that are currently running.
 
-1. Use the `stop-task` command to stop all the replication tasks that are currently running.
 2. Use DM-Ansible to [stop the entire DM cluster](deploy-a-dm-cluster-using-ansible.md#step-10-stop-the-dm-cluster).
+
 3. Manually clean up the relay log directory of the DM-worker corresponding to the MySQL master whose binlog is reset.
```
Comment (ran-huang) [inline en/faq.md:79]: @csuzhangxc This MySQL `master` should be replaced by another word. Please help confirm if my suggestion is technically correct.
```suggestion
3. Manually clean up the relay log directory of the DM-worker corresponding to the MySQL cluster whose binlog is reset.
```

**Pull request merged**